tetris/board.go

```diff
@@ -6,17 +6,17 @@ package tetris
 // The current moving piece is overlay on top of the Board until it is emprinted
 // on the Board.
 type Board struct {
-	m    [][]int
-	w, h int
-	pieceColors[][]string
+	m           [][]int
+	w, h        int
+	pieceColors [][]string
```

Comment:
user the new type `Color`. 

`pierceColors [][]Color`. In this way we are clear that no every string can be passed here, but only `Color`.